examples/linear_model/plot_orthogonal_regression_nonanalytic_behavior.py

```diff
@@ -0,0 +1,130 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from skcosmo.linear_model import OrthogonalRegression
+
+# Example that shows how the `project_to_prediction_space` option in :class:`skcosmo.linear_model.OrthogonalRegression` can result in nonanalytic behavior
```

Comment:
This example on a whole is not nearly descriptive to be useful to users. From the code I've gathered that you want to compare a prism to a cube using orthogonal regression? If that's so the misnaming will be incredibly confusing. Please rewrite (better yet as a jupyter notebook) with a bit more attention to how the reader will use this.